tests/dummy/app/pods/view/renderers/views/checkbox-array.js

```diff
@@ -3,12 +3,54 @@ export default {
     main: {
       children: [
         {
-          model: 'severity',
-          renderer: {
-            name: 'checkbox-array'
-          }
+          children: [
+            {
+              model: 'severity',
+              renderer: {
+                name: 'checkbox-array'
+              }
+            }
+          ],
+          label: 'Severity',
+          collapsible: true
+        },
+        {
+          children: [
+            {
+              model: 'serviceAffecting',
+              renderer: {
+                name: 'checkbox-array',
+                data: [
```

Comment:
Rather than this being an array of key/value pairs, I'd suggest a hash that maps the already defined `enum` values from the model to user-friendly labels:

```js
labels: {
  'NOT_SERVICE_AFFECTING': 'No',
  'SERVICE_AFFECTING': 'Yes'
}
```